src/sign/index.ts

```diff
@@ -52,13 +53,14 @@ export default class Client {
   async sign(web3: Web3Provider | Wallet, address: string, message, types) {
     // @ts-ignore
     const signer = web3?.getSigner ? web3.getSigner() : web3;
-    if (!message.from) message.from = address;
+    const checksumAddress = getAddress(address.toLowerCase());
```

Comment:
Why toLowercase here?

Reply:
I used the lowercase function for basic error checking in case we receive a malformed checksum address

Reply:
In which case does it throw and error here? I dont think toLowercase change anything here

Reply:
Yeah it's probably a good idea if we throw an error if we get a bad checksum address. I'll get rid of it